### PR TITLE
12 cl footer - Added code footer in index.html and general css

### PR DESCRIPTION
**Pull request by idacarlin95**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,6 +19,32 @@ <h2>KEEPERS</h2>
...
+    <footer>
+        <div class="footer-content">
+            <div class="footer-section socialmedia">
+                <div class="social-icons">
+                <a href="#"><img src="../australia-zoo/images/icons/github.png" alt="github-icon"></a>
+                <a href="#"><img src="../australia-zoo/images/icons/instagram.png" alt="instagram-icon"></a>
+                <a href="#"><img src="../australia-zoo/images/icons/national-georaphic.png" alt="national-georaphic-icon"></a></div>
+                <div class="footer-bottom">
+                    <p>Copyright 2024 - The Zookeepers</p>
+                </div>
```
Comment (jasminedns) [inline index.html]: need some indentation, also please make a ticket to decide which links are we going to use and to put the links once we decided!

File: styles/general.css [hunk 2]
```diff
@@ -40,4 +58,127 @@ header {
...
+.footer-section.partners {
+    display: flex;
+    flex-direction: column;
+    font-family: 'KoHoregular';
+    font-size: var(--small-font);
+    justify-content: space-between;
+    text-align: center;
+    align-items: center;
+    height: 160px;
+    border-left: 1px solid #fff;
```
Comment (jasminedns) [inline styles/general.css]: change the color to a variable

**Review by jasminedns — CHANGES_REQUESTED** [review]
besides these little things looks good!

**Review by jasminedns — APPROVED** [review]
looking good!

**Review by TobiasKjernell — APPROVED** [review]
looked good with the changes!

**Pull request merged**